Disable tests in FLTK because they use the fluid tool on the host.

Patch borrowed from OpenEmbedded.

Index: fltk-1.1.9/Makefile
===================================================================
--- fltk-1.1.9.orig/Makefile	2008-06-06 16:14:11.936736201 +0200
+++ fltk-1.1.9/Makefile	2008-06-06 16:14:22.526357637 +0200
@@ -27,7 +27,7 @@
 
 include makeinclude
 
-DIRS	=	$(IMAGEDIRS) src fluid test documentation
+DIRS	=	$(IMAGEDIRS) src fluid documentation
 
 all: makeinclude fltk-config
 	for dir in $(DIRS); do\
